src/config/mod.rs

```diff
@@ -37,7 +39,8 @@ pub struct Config {
     pub favicon: Option<String>,
     pub path_prefix: Option<String>,
     pub license: Option<String>,
-    pub md_book: Option<String>,
+    /// The config for using mdbook
+    pub md_book: Option<BookConfig>,
```

Comment:
i would rename this MdBookConfig as i suspect in the fullness of time we may support other "book" providers